src/megaapi_impl.cpp

```diff
@@ -10674,11 +10671,9 @@ MegaNodeList* MegaApiImpl::searchInAllShares(const char *searchString, MegaCance
     else
     {
         // Search in public links
-        ::mega::unique_ptr<MegaNodeList> publicLinks(getPublicLinks(MegaApi::ORDER_NONE));
-
-        for (int i = 0; i < publicLinks->size() && !(cancelToken && cancelToken->isCancelled()); i++)
+        for (const auto& item : client->mPublicLinks)
```

Comment:
So we removed the feature to cancel the ongoing search? You need to keep that condition!

Reply:
Bad copy paste so sorry.